graspy/embed/ase.py

```diff
@@ -57,12 +63,24 @@ class AdjacencySpectralEmbed(BaseEmbed):
         'truncated'. The default is larger than the default in randomized_svd 
         to handle sparse matrices that may have large slowly decaying spectrum.
 
-    check_lcc : bool , optional (defult = True)
+    check_lcc : bool , optional (default = True)
         Whether to check if input graph is connected. May result in non-optimal 
         results if the graph is unconnected. If True and input is unconnected,
         a UserWarning is thrown. Not checking for connectedness may result in 
         faster computation.
 
+    diag_aug : bool, optional (default = True)
+        Whether to replace the main diagonal of the adjacency matrix with a vector 
+        corresponding to the degree (or sum of edge weights for a weighted network) 
+        before embedding. Empirically, this produces latent position estimates closer
+        to the ground truth. 
+
+    ptr : bool, optional (default = True) 
```

Comment:
1) I dont agree that PTR should be part of ASE.
2) PTR doesn't work for graphs with negative weights
3) If we do include it, I think it should allow for all three options and off by default.